.travis.yml

```diff
@@ -0,0 +1,6 @@
+language: node_js
```

Comment:
not related, but setting sudo to false [will speedup build times by using docker](http://blog.travis-ci.com/2014-12-17-faster-builds-with-container-based-infrastructure/), see [the zamboni PR](https://github.com/mozilla/zamboni/pull/2821) for an example, or a [simpler one for the readinglist](https://github.com/mozilla-services/readinglist/commit/85dd328d2600909b72edbcb93d6c8194d01e9b5a)